src/net_processing.h

```diff
@@ -72,6 +72,8 @@ class PeerLogicValidation final : public CValidationInterface, public NetEventsI
     /** If we have extra outbound peers, try to disconnect the one with the oldest block announcement */
     void EvictExtraOutboundPeers(int64_t time_in_seconds) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
 
+    // void EraseTxRequest(uint256& txid) EXCLUSIVE_LOCKS_REQUIRED(cs_main) { m_already_asked_for.erase(txid); }
```

Comment:
nit: Looks like you can remove these two lines now

Reply:
oops!